src/status_im/utils/handlers.cljs

```diff
@@ -29,9 +29,9 @@
     :after
     (fn check-handler
       [context]
-      (let [new-db (get-coeffect context :db)
+      (let [new-db (get-effect context :db)
```

Comment:
I knew that they are working incorrectly and that the information is meaningless, but did not know why